cerberus-cli/src/main/java/ca/on/oicr/gsi/cerberus/cli/TabReportGenerator.java

```diff
@@ -455,7 +454,7 @@ protected String apply(SampleProvenance lims) {
     cs.add(record.provider());
     cs.add(record.lims().getProvenanceId());
     cs.add(record.lims().getVersion());
-    cs.add(record.lims().getLastModified().toString());
+    cs.add(record.lims().getLastModified().toInstant().atZone(ZoneOffset.UTC).toString());
```

Comment:
You might want to create your own `DateTimeFormatter` to ensure you get the format you want as the `Instant#toString` implementation may not be consistent. It should use an ISO-8601 formatter, but both formats you mentioned are ISO-8601 compliant, and the JavaDoc even suggests it will output the one that you _don't_ want